keyboards/massdrop/alt/keymaps/xulkal/keymap.c

```diff
@@ -77,3 +77,82 @@ bool process_record_keymap(uint16_t keycode, keyrecord_t *record)
 
     return true;
 }
+#include QMK_KEYBOARD_H
```

Comment:
`make massdrop/alt:xulkal` does not build.

Looks like you might have a bad merge or something here as there is duplicate stuff like includes, keymap object, defines, etc

Reply:
bah, fixed